README.md

```diff
@@ -503,25 +592,28 @@ The following UCAN fragment would be valid to invoke as `did:key:zH3C2AVvLMv6gmM
+Each capability MUST either be originated by the issuer (root capability, or "parenthood") or have one-or-more proofs in the `prf` field to attest that this issuer is authorized to use that capability ("introduction"). In the introduction case, this check MUST be recursively applied to its proofs until a root proof is found (i.e. issued by the resource owner).
 
-Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its witnesses. The time bounds MUST also be equal to or contained inside the time bounds of the witnesses' time bounds. This lowering of rights at each delegation is called "attenuation."
+Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its proofs. The time bounds MUST also be equal to or contained inside the time bounds of the proof's time bounds. This lowering of rights at each delegation is called "attenuation."
 
 ## 5.4 Rights Amplification
 
-Some capabilities are more than the sum of their parts. The canonical example is a can of soup and a can opener. You need both to access the soup inside the can, but the can opener may come from a completely separate source than the can of soup. Such semantics MAY be implemented in UCAN capabilities. This means that validating particular capabilities MAY require more than one direct witness. The relevant witnesses MAY be of a different resource and action from the amplified capability. The delegated capability MUST have this behaviour in its semantics, even if the witnesses do not.
+Some capabilities are more than the sum of their parts. The canonical example is a can of soup and a can opener. You need both to access the soup inside the can, but the can opener may come from a completely separate source than the can of soup. Such semantics MAY be implemented in UCAN capabilities. This means that validating particular capabilities MAY require more than one direct proof. The relevant proofs MAY be of a different resource and action from the amplified capability. The delegated capability MUST have this behaviour in its semantics, even if the proofs do not.
 
 ## 5.6 Content Identifiers
 
-UCANs MAY be referenced by content ID (CID), per the [multiformats/cid](https://github.com/multiformats/cid) specification. The resolution of these addresses is left to the implementation and end-user, and MAY (non-exclusively) includes the following: local store, a distributed hash table (DHT), gossip network, or RESTful service.
+A UCAN token MUST be referenced as [CIDv1](https://docs.ipfs.io/concepts/content-addressing/#identifier-formats) with the following configuration:
+* CID Version: `1`
+* Multibase: [`b` base32](https://github.com/multiformats/multibase/blob/master/multibase.csv#L12)
+* Multicodec: [`0x55` raw data](https://github.com/multiformats/multicodec/blob/master/table.csv#L39)
```

Comment:
That's a really good point!

@Gozala: you're proposing the use of `dag-json` for the deterministic codec, yes?

Reply:
> Would this explicitly deny using dag-ucan in valid UCANs?

It does sound like it would indeed make [dag-ucan](https://github.com/ipld/js-dag-ucan) incompatible with a spec as it uses CBOR encoding with a following schema https://github.com/ucan-wg/spec/pull/67, when possible.

@expede is this intentional ? Otherwise can we either include CBOR variant or at least change wording so that RAW isn't the only valid option.

Reply:
> @Gozala: you're proposing the use of dag-json for the deterministic codec, yes? IIRC dag-ucan didn't go forward.

I'm not sure I fully understand that question, I think there are couple of things mixed up, so I"ll try to clarify:

1. dag-ucan did go forward and it uses CBOR encoding when possible (that is turning that CBOR to JSON and then formatting as JWT produces same signature), and RAW encoding otherwise.
2. Idea of introducing dag-ucan multicodec code was dropped in favor of just CBOR because:
    1. It did not provide a lot of benefit, mostly signaling in links
    2. And came with tradeoff that required registering new codecs even though it is CBOR with some schema.
3. I have been advocating for ucan implementations using `dag-json` as that would allow them to be representation agnostic (could be encoded in CBOR  without signature changes). However that is unrelated to dag-ucan.

Reply:
@Gozala There was a previous version that included the `dag-ucan` multicodec. When that PR was nixed, I dropped its prefix from this document. I just forgot to update this section after dag-json was settled on. I'll add that now 👍

Reply:
@Gozala any concern with limiting it to CIDv1 + base32?